src/locale/misc/smartSearch/en.yml

```diff
@@ -143,7 +143,7 @@ task:
     taskStatusSelect:
         completed: completed
         ignored: ignored
-        assigned: assigned to
+        assigned: are assigned to
```

Comment:
This should be "been assigned".
![image](https://user-images.githubusercontent.com/550212/150640385-25d259ee-4bfd-4588-8119-94c8f85b7c50.png)

Reply:
fixed